controllers/sriov-controller/service-controller.go

```diff
@@ -1,17 +1,3 @@
-// Copyright (c) 2018 Cisco and/or its affiliates.
```

Comment:
Unsure why the copyright is removed here as well.

Reply:
restored